### PR TITLE
FinalPlace controller and form

### PR DESCRIPTION
**Pull request by JitkaCNGroup**

### CODE REVIEW

File: src/main/java/dk/cngroup/lentils/controller/FinalPlaceController.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@Controller
+@RequestMapping("/finalplace")
+public class FinalPlaceController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(FinalPlaceController.class);
+    private final String VIEW_FINALPLACE_FORM = "finalplace/form";
```
Comment (crlikcngroup) [inline src/main/java/dk/cngroup/lentils/controller/FinalPlaceController.java]: Tyhle konstanty taky nemuzou byt static?

File: src/main/java/dk/cngroup/lentils/controller/FinalPlaceController.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@Controller
+@RequestMapping("/finalplace")
+public class FinalPlaceController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(FinalPlaceController.class);
```
Comment (mjanousek) [inline src/main/java/dk/cngroup/lentils/controller/FinalPlaceController.java]: Pouziva se ten logger?

File: src/main/java/dk/cngroup/lentils/service/FinalPlaceService.java [hunk 3]
```diff
@@ -22,8 +23,16 @@ public FinalPlace save(FinalPlace finalPlace) {
...
     }
 
-    public Optional<FinalPlace> get(Long id) {
-        return finalPlacerepository.findById(id);
+    public FinalPlace get() {
+        List<FinalPlace> finalPlaces = getAll();
+        if (finalPlaces.size() == 0 ){
+            return new FinalPlace();
+        }
+        return finalPlaces.get(0);
```
Comment (mjanousek) [inline src/main/java/dk/cngroup/lentils/service/FinalPlaceService.java]: Co kdyz to vrati vic mist z DB?

File: src/main/java/dk/cngroup/lentils/service/FinalPlaceService.java [hunk 4]
```diff
@@ -22,8 +23,16 @@ public FinalPlace save(FinalPlace finalPlace) {
         return finalPlacerepository.save(finalPlace);
     }
 
-    public Optional<FinalPlace> get(Long id) {
-        return finalPlacerepository.findById(id);
+    public FinalPlace get() {
+        List<FinalPlace> finalPlaces = getAll();
+        if (finalPlaces.size() == 0 ){
```
Comment (crlikcngroup) [inline src/main/java/dk/cngroup/lentils/service/FinalPlaceService.java]: Jen format prosim.

File: src/main/resources/templates/finalplace/form.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="cz"
+      xmlns:th="http://www.springframework.org/schema/data/jaxb">
+<head>
+    <meta charset="UTF-8">
+    <title>FinalPlace</title>
+</head>
+<body>
+<h2>"Poslední" místo<h2>
```
Comment (mjanousek) [inline src/main/resources/templates/finalplace/form.html]: Misto zakonceni

File: src/main/java/dk/cngroup/lentils/service/FinalPlaceService.java (hunk 4 above)
Comment (JitkaCNGroup) [reply src/main/java/dk/cngroup/lentils/service/FinalPlaceService.java]: Done

File: src/main/java/dk/cngroup/lentils/controller/FinalPlaceController.java (hunk 2 above)
Comment (JitkaCNGroup) [reply src/main/java/dk/cngroup/lentils/controller/FinalPlaceController.java]: Casem, treba ..

File: src/main/java/dk/cngroup/lentils/service/FinalPlaceService.java (hunk 3 above)
Comment (JitkaCNGroup) [reply src/main/java/dk/cngroup/lentils/service/FinalPlaceService.java]: nevrati, jak by se tam dostaly ?

File: src/main/java/dk/cngroup/lentils/controller/FinalPlaceController.java (hunk 2 above)
Comment (mjanousek) [reply src/main/java/dk/cngroup/lentils/controller/FinalPlaceController.java]: Smazat

Comment (JitkaCNGroup) [reply src/main/java/dk/cngroup/lentils/controller/FinalPlaceController.java]: Dona :-(

File: src/main/resources/templates/finalplace/form.html (hunk 5 above)
Comment (JitkaCNGroup) [reply src/main/resources/templates/finalplace/form.html]: Done

**Pull request merged**